samples/kubeflow/README.md

```diff
@@ -65,21 +112,59 @@ TODO: currently the policies are hard coded with the OPA deployment
 
 ## Data user instructions
 
-Port forward the istio ingress gateway to be able to open Kubeflow home.
 
+1. Create a port-forward to communicate with Kubeflow:
 ```bash
-kubectl port-forward svc/istio-ingressgateway -n istio-system 8080:80
+kubectl port-forward svc/istio-ingressgateway -n istio-system 8080:80 &
+
+cd samples/kubeflow/
 ```
 
-Then open you browser in "http://localhost:8080". Click **Start Setup** and then **Finish**. Then, click **Notebook Servers**. In the notebooks page select in the top the anonymous namespace and then click **New Server**.
+2. Upload the notebook:
+- Open your browser in `http://localhost:8080`.
+- Click **Start Setup** and then **Finish** (use the `anonymous` namespace).
+- Click **Notebook Servers** (in the left).
+- In the notebooks page select in the top left the `anonymous` namespace and then click **New Server**.
+- In the notebook server creation page, set `kf-notebook` in the **Name** box and then click **Launch**. Wait for the server to become ready.
+- Click **Connect** and upload `kfM4DPolicyDemo.ipynb` notebook to the server.
```

Comment:
can we rename the notebook name: Demo-->Sample ?

Reply:
Done